presets/tests/201-wavecode.milk

```diff
@@ -1,6 +1,6 @@
 [preset00]
-per_frame_1000=// use wavecode
```

Comment:
why>

Reply:
I was just trying to find a way to add comments that seemed to work.  I didn't realize that full line comments also seem to work.